src/opendune.c

```diff
@@ -21,6 +21,9 @@
 #include <time.h>
 #if defined(WITH_SDL) || defined(WITH_SDL2)
 #include <SDL.h>
+#ifdef _WIN32
+#undef main
```

Comment:
This will avoid to have to set /SUBSYSTEM:CONSOLE or /SUBSYSTEM:WINDOW on VS Linker System options...